lib/helium/sensor.rb

```diff
@@ -59,8 +53,7 @@ def as_json
         name: name,
         mac: mac,
         ports: ports,
-        last_seen: last_seen,
-        labels: labels
```

Comment:
is anything using this part of the json? i forget why it was in here originally. but agree that it makes sense to keep it separate